firmware/console/ethernet_console.cpp

```diff
@@ -81,23 +81,6 @@ struct EthernetThread : public TunerstudioThread {
 static EthernetThread ethernetConsole;
 
 void startEthernetConsole() {
-#ifndef STM32H7
-	// TODO: why does this break H7? I thought the pins were the same?
-	efiSetPadMode("ETH_RMII_REF_CLK",  Gpio::A1, PAL_MODE_ALTERNATE(0xb));
```

Comment:
you've killed my fancy pin names

not that those are really needed

Reply:
those fancy names use extra flash memory 🙂